src/adapters/env.ts

```diff
@@ -29,14 +53,23 @@ const envToDomain = ({
   VITE_FIAT_EXCHANGE_RATES_API_KEY,
   VITE_FIAT_EXCHANGE_RATES_API_URL,
   VITE_FIAT_EXCHANGE_RATES_ETHEREUM_USDC_ADDRESS,
+  VITE_OUTDATED_NETWORK_MESSAGE,
+  VITE_OUTDATED_NETWORK_TITLE,
+  VITE_OUTDATED_NETWORK_URL,
   VITE_POLYGON_ZK_EVM_BRIDGE_CONTRACT_ADDRESS,
   VITE_POLYGON_ZK_EVM_EXPLORER_URL,
   VITE_POLYGON_ZK_EVM_NETWORK_ID,
   VITE_POLYGON_ZK_EVM_RPC_URL,
   VITE_USE_FIAT_EXCHANGE_RATES,
 }: Env): Promise<domain.Env> => {
   const polygonZkEVMNetworkId = z.number().positive().parse(Number(VITE_POLYGON_ZK_EVM_NETWORK_ID));
-  const useFiatExchangeRates = z.boolean().parse(VITE_USE_FIAT_EXCHANGE_RATES === "true");
+  const useFiatExchangeRates = stringBooleanParser.parse(VITE_USE_FIAT_EXCHANGE_RATES);
+  const bridgeApiUrl = VITE_BRIDGE_API_URL;
+  const outdatedNetwork: domain.Env["outdatedNetwork"] = {
+    message: VITE_OUTDATED_NETWORK_MESSAGE || undefined,
+    title: VITE_OUTDATED_NETWORK_TITLE || undefined,
+    url: VITE_OUTDATED_NETWORK_URL || undefined,
```

Comment:
These variables are already `string | undefined`

Reply:
Right, I've removed this check from here.